File: MainApp/src/main/java/com/sleticalboy/dailywork/csv/PathActivity.java
package com.sleticalboy.dailywork.csv;

import android.view.View;

import com.sleticalboy.dailywork.R;
import com.sleticalboy.dailywork.base.BaseActivity;
import com.sleticalboy.weight.view.TestView;

/**
 * Created on 18-3-1.
 *
 * @author leebin
 * @version 1.0
 */
public class PathActivity extends BaseActivity {

    @Override
    protected void initData() {
    }

    @Override
    protected void initView() {
        TestView testView = findViewById(R.id.test);
        final View singleLine = findViewById(R.id.singleLine);
        // singleLine.setOnClickListener(v -> testView.postDelayed(() -> {
        //     testView.update();
        //     singleLine.performClick();
        // }, 250L));
        singleLine.setOnClickListener(v -> {
            testView.invalidate();
        });
    }

    @Override
    protected int layoutResId() {
        return R.layout.activity_path;
    }
}

File: MainApp/src/main/java/com/sleticalboy/learning/csv/PathActivity.kt
package com.sleticalboy.learning.csv

import android.view.View
import com.sleticalboy.learning.R
import com.sleticalboy.learning.base.BaseActivity
import com.sleticalboy.weight.view.TestView


/**
 * Created on 18-3-1.
 *
 * @author leebin
 * @version 1.0
 */
class PathActivity : BaseActivity() {

    override fun initData() {}

    override fun initView() {
        val testView = findViewById<TestView>(R.id.test)
        val singleLine = findViewById<View>(R.id.singleLine)
        // singleLine.setOnClickListener(v -> testView.postDelayed(() -> {
        //     testView.update();
        //     singleLine.performClick();
        // }, 250L));
        singleLine.setOnClickListener { testView.invalidate() }
    }

    override fun layoutResId(): Int {
        return R.layout.activity_path
    }
}